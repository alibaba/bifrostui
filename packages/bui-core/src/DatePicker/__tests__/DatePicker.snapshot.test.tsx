import { snapshotTest } from 'testing';

describe('DatePicker demo snapshot', () => {
  snapshotTest('DatePicker');
});
