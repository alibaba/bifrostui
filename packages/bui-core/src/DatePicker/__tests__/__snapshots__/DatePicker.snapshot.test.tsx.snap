// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker demo snapshot DatePicker demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
    onClick={[Function]}
    type="button"
  >
    2025/3/6
  </button>
</div>
`;

exports[`DatePicker demo snapshot DatePicker demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
    onClick={[Function]}
    type="button"
  >
    2025/3/6
  </button>
</div>
`;

exports[`DatePicker demo snapshot DatePicker demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <button
    className="bui-btn bui-btn-outlined bui-btn-medium bui-btn-default"
    onClick={[Function]}
    type="button"
  >
    2025/3/6
  </button>
</div>
`;
